### ISSUE TITLE
more flavor

### ISSUE THREAD

**Issue by marcelotc003** [SlimeKnights/TinkersConstruct#5144]
### Minecraft Version

1.18.

### Forge Version

40.2.9

### Mantle Version

1.9.45

### Tinkers' Construct Version

3.6.4.113

### Problem description

Hi, so i think that the mod may use more weapons, more flavor. I read what de tinkers 3 would be removing and i thought that I would still use a longsword even if it is just a sword with diferent stats you see?. So for a "problem" description I would say that the mod could use more flavor. I am almost 100% sure you guys think about adding more stuff constantly, so in this post i am going to sugest how to implement (as in balance)
This will be just sugestions to what add to the mod and it may seen strange, but that is just what I would do if I could do something. Thanks for your time.

### Suggested solution

  If the "problem" is the lack o flavor, just add more. I would love to see more sword variants, the rapier, a longsword, maybe some diferent weapons like a Macuahuitl, a saber, a mace, a club, a katana, and that is just for the weapons. As for ranged weapons I like the two options we have, but it would be nice if the mod added arrows and bolts of its own, with different materials giving different effects (like iron penetrating armor, bronze giving more damage to un-armored targets, stone dont need to have any special stuff, just being easier to find is a big + in my opinion), on second thought, it would be nice to add a Jazail in the mod, it would be a single shot, slow reload and veary powerfull gun that shoots nugget like projectile (I would use it a lot) As for tools we have a pretty wide array of choices and i am satisfied. 
  For in game balance of my sugestions, the rapier, the longword and the broad sword can come back as they were. The Macuahuitl can give a base extra damage against un-aromored victims and a slow swing, a saber can be fast with a increased chance for critcal hits, the mace can have more damage against armored targets, the club will slow the target, and the katana can start with a beheading efect.

### Alternatives considered

As an alternative you can add more shields and armor to the mod, also counts as flavor (although the first one is better in my opinion).

### Searched for existing enhancement?

Used the search bar, Checked the FAQ

**Comment by KnightMiner** [comment]
This suggestion is both more thought out than the average suggestion, but also not thought out enough.

You keep suggesting "add more things" and "add back things that used to exist" but never once stopped to ask "why were those things removed?"

Flavor is not about having more things. I could add 1000 sword variants and the mod would be worse because of it. Flavor is about adding interesting and useful things. I personally find it far more interesting for someone to create unique weapons, such as a creeper killing pickaxe or the excavator of extreme knockback, rather than just crafting a third clone of a sword with a different name. Every harvest tool is also a weapon, each has a specialty, you just have to put a *tiny* amount of creative effort into using it. You have 15 choices for melee weapons, you just never considered looking past swords. Its not the mod that lacks flavor, its your choices in the weapons you build that do.

----

Now, if you really want more weapons that are just clones of existing weapons, let me introduce you to the mod Json Things. You can by writing Thing Packs (which work like datapacks) add as many new tool variants as your heart desires. The mod Tinkers' Json Things serves as an example. Go wild, design tools to your heart's content.

----

To answer the question you should have asked but never did, all the removed weapons have good reasons to be removed.

* The longsword was just a sword with different stats and a nonsensical ability for a sword. The ability was moved to a slimesling, which will eventually end up as a tool modifier.
* The rapier was just stupidly OP. The vein hammer and the pickaxe are both a less OP variant of the high piercing weapon, while daggers are the high speed weapon
* Modifiable ammo is just utterly broken as a concept. Tinker Tools are unique and expensive creation, ammo is disposable. And the design from 1.12 and before for ammo was so awkward that you have no idea whether a given modifier goes on your bow or arrow. I refuse to bring back a bad design just because your taste buds are not satisfied.
* The broadsword still exists. Its just called "sword" now. The fact you did not realize its the same makes it clear you did not care about the behavior of the tools, you just wanted options.

As for your other suggestions, sound like great modifier or material trait ideas, not brand new tools. A sword that does slightly different damage against different enemy types is how modifiers like smite or cooling work. It is not interesting enough to make a brand new tool and confuse players. Your ideas included:

* More damage against armored: that is called piercing, use a pickaxe or vein hammer
* More damage against unarmored: not convinced its needed, just use sharpness
* Increased chance for critical hits: critical hits are not chance based, they are triggered by falling. You already have 100% chance.
* Slowing the target: we have a modifier for that, called slowing.
* Sword starting with beheading: exists, called the cleaver.
* As for guns: no

----

tl;dr: Adding more stuff is not flavor. You are free to make your own addon to add more stuff if you disagree.
